holoviews/core/util.py

```diff
@@ -703,7 +705,7 @@ def isfinite(val):
         if val.dtype.kind in 'USMO':
             return np.ones_like(val, dtype=bool)
         return np.isfinite(val)
-    elif isinstance(val, datetime_types+(basestring,)):
+    elif isinstance(val, datetime_types+timedelta_types+(basestring,)):
```

Comment:
Is this really an ``isfinite`` fix?

Reply:
Yes, it's in the ``isfinite`` function isn't it?

Reply:
Sorry, the Github diff was misleading! Isn't there another recent issue about ``pd.Period``?

Reply:
Yes, but that's also added to datetime_types to exclude it from isfinite checks.